### PR TITLE
new icon: okta (original, original-wordmark, plain, plain-wordmark)

### LINKED ISSUE #171: Icon Request: OKTA
Can you please add the icons for Okta (authentication)?

I have added the available files from their website to this issue:

[Download all Okta Developer Logos](https://www.okta.com/sites/default/files/okta_developer_logos.zip)

### PR DESCRIPTION
**Pull request by weh**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**
closes #171

## Link to prove your SVG is correct and up-to-date.

https://www.okta.com/press-room/media-assets/developer/

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -5194,6 +5194,27 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "okta",
+        "altnames": [
+            "okta developer"
+        ],
+        "tags": [
+            "auth"
```
Comment (Panquesito7) [inline devicon.json:5203]: What do you think about this?
```suggestion
            "auth",
            "security"
```

Comment (weh) [reply devicon.json:5203]: sure, makes sense

**Review by Panquesito7 — DISMISSED** [review]
Looks great. Thanks! 🚀

**Review by Snailedlt — CHANGES_REQUESTED** [review]
@weh Isn't okta a company, and not a product?
From looking at their website I think the company is called okta, and they have multiple products:
image

If this is the case, `okta` is out of scope for devicons
ref: [What Icons Do We Accept?](https://github.com/devicons/devicon/wiki/What-Icons-Do-We-Accept%3F)

**Comment by weh** [comment]
Hmm, i think i saw Okta as Auth provider Service. 
But if you do not want those Icons, you should IMHO triage the Icon requests, before the work is done and the MR exists. I was especially looking for older Issues, that exist for a longer time, not the newest ones in the list. 
Maybe you should clarify the triage process and add some kind of label, that clearly shows that this icon is going to be accepted, or close the open Issue.

**Comment by Snailedlt** [comment]
Agreed, we should definitely do that. In fact I did go through all the old issues a while back and triaged a few of them. As you can imagine though, it takes a long time to look through all of them thoroughly when there are 200+ issues, so many of them were just glanced over.

Sadly we are too few (2) active maintainers atm, so we prioritize PRs and automation issues over all else.

All that said, I'm not convinced on adding or not adding this icon yet, so please do make some arguments for/against adding it :)

**Comment by Snailedlt** [comment]
@Panquesito7 Could you also take another look at this, considering the things said by me and weh above?

**Comment by Snailedlt** [comment]
Well, I did a bit more research... and I couldn't find any logos for their products. Their product names are really generic too, and aren't really product names, but just descriptions of whta the products do. So it looks like they're kinda marketing okta as a platform and not a company logo... in which case I would put it in the same category as Amazon Web Services, which means it's eligible.

I'll remove the awaiting triage label then, so it's ready for review :)
Thanks for your patience @weh .
And thanks for your input @lunatic-fox :)

**Review by Snailedlt — APPROVED** [review]
LGTM! ✔️ 

I made a commit to unify the paths for the plain versions, so that it complies with our [SVG-Standards](https://github.com/devicons/devicon/wiki/SVG-Standards).

Thank you for your contribution, and your patience! 
It's always a pleasure to review your PR's @weh 💯

**Comment by Snailedlt** [comment]
I'll just merge this right away since a previous version was approved by @Panquesito7 :)

**Pull request merged**